pages/docs/installation/installation-packages.md

```diff
@@ -42,7 +42,21 @@ These packages are maintained by the preCICE community and may be occasionally o
 However, we appreciate the effort and you may be able to contribute to them.
 
 - [MSYS2](https://packages.msys2.org/base/mingw-w64-precice) (for Windows, built with MinGW), [thread on our forum](https://precice.discourse.group/t/precice-and-mingw-packages/382)
-- [Nix / NixOS](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&query=precice)
 - [EasyBuild](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/p/preCICE)
 - [Conda](https://anaconda.org/conda-forge/precice) (see also packages [pyprecice](https://anaconda.org/conda-forge/pyprecice) and [fenicsprecice](https://anaconda.org/conda-forge/fenicsprecice))
 - [FreeBSD](https://www.freshports.org/science/precice)
+
+### Nix / NixOS
+
+In addition to the community efforts listed above, preCICE and several of the bindings and adapters are available for Nix in the [precice/nix-packages repository](https://github.com/precice/nix-packages/).
+
+If you are using Nix (or NixOS for that matter), you can simply run the command `nix shell github:precice/nix-packages#precice-aster github:precice/nix-packages#precice-openfoam-adapter` for instance, which will drop you into a shell with code-aster and the OpenFOAM adapter accessible.
```

Comment:
A more useful example would be OpenFOAM and CalculiX, as this is used in several of our tutorials.